src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/request/TimeSlot.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request
+
+import java.time.LocalTime
+
+enum class TimeSlot(private val predicate: (LocalTime) -> Boolean) {
+  AM({ t -> t.hour < 12 }),
+  PM({ t -> t.hour in 12..16 }),
+  ED({ t -> t.hour >= 17 }),
```

Comment:
Thought 18pm + was ED - but meh..

Reply:
I took these times from prison-api.  That is not to say we can't have 18.